website/content/labs/submit.md

```diff
@@ -2,22 +2,6 @@
 title: "How to submit your labs"
 date: 2018-01-28
 publishdate: 2018-01-28
-draft: true
+draft: false
 ---
 
-You will be submitting your labs using a tool called `chisubmit` that is available on the CS machines. Before using this tool, you first need to enable it inside your local repository. Just go into your local repository directory, and run the following:
```

Comment:
is this whole file effectively blank now? also are they not using `chisubmit`?

Reply:
Github classroom paired with gradescope is the new kid in town

Reply:
This file probably won't be ready before the lab is published (I still have to do some further testing with Gradescope+GitHub) but, yeah, basically they will be using Gradescope to submit their labs.